cypress/integration/note/create.spec.js

```diff
@@ -0,0 +1,114 @@
+import { v4 } from 'uuid';
+import awsConfig from '../../support/config/aws.config';
+
+const NOTE_CONTENT = `Note - ${v4()}`;
+
+describe('Note - Create', () => {
+  const createdNotes = [];
+
+  beforeEach(() => {
+    cy.visit('/');
+    const mail = Cypress.env('mail');
+    const pass = Cypress.env('pass');
+    cy.signIn(mail, pass);
+    cy.visit('/notes/new');
+  });
+
+  it('should have a note creation page', () => {
+    cy.get('#content').should('have.length', 1);
+    cy.get('#file').should('have.length', 1);
+    cy.get('#uploadFileButton').should('have.length', 1);
+    cy.get('#createButton').should('have.length', 1);
+  });
+
+  it("should disable create button if there's no note given", () => {
```

Comment:
Looks like prettier did not format this page.

Reply:
Will look into it, thanks!

Reply:
We can run prettier across the whole project in a new PR with logic changes to clean it up.

Reply:
Yep, will create a branch for clean-ups after we closed this branches.